README.md

```diff
@@ -75,6 +76,25 @@ it, simply add the following line to your Podfile:
 pod 'Mute'
 ```
 
+### Carthage
+
+[Carthage](https://github.com/Carthage/Carthage) is a decentralized dependency manager that builds your dependencies and provides you with binary frameworks.
+
+You can install Carthage with [Homebrew](http://brew.sh/) using the following command:
+
+```bash
+$ brew update
+$ brew install carthage
+```
+
+To integrate Alamofire into your Xcode project using Carthage, specify it in your `Cartfile`:
```

Comment:
```suggestion
To integrate Mute into your Xcode project using Carthage, specify it in your `Cartfile`:
```

Reply:
@akramhussein 
that is mistake caused by copy paste, I will fix it.